level/ui.gd

```diff
@@ -95,3 +95,4 @@ func show_score():
 
 func _on_restart_pressed():
 	global.restart()
+	Input.set_mouse_mode(Input.MOUSE_MODE_CONFINED)
```

Comment:
This should be done in `global.restart()`.

Reply:
Or maybe not, lemme think.

Reply:
This should actually not be necessary, `restart()` reloads the scene and thus the mouse mode should be confined in `_ready`.

Reply:
I tested without it, the mouse doesn't confine itself unless I include this line.